pghoard/rohmu/object_storage/azure.py

```diff
@@ -353,5 +377,9 @@ def get_or_create_container(self, container_name):
             self.conn.create_container(container_name)
         except ResourceExistsError:
             pass
+        except HttpResponseError:
```

Comment:
I don't know how Azure authentication works, but if we are using a service principal can we just skip this step ? Otherwise we could ignore an error when it is expected to actually create the container.

Reply:
Definitely looks like we should check that the error was indeed related to access restricting and log (debug level?) notification about assuming the bucket exists.

Reply:
Finer grained error handling here absolutely makes sense. 

Note: The HTTP request made by `azure.storage.blob.baseBlobService.create_container()` does not populate the `error` attribute of the `HttpResponseError` class, so I had to resort to parsing the `exc_msg` attribute of its `AzureError` base class.